src/cpu/cpu-command.cpp

```diff
@@ -64,6 +64,7 @@ class CommandExecutor
     void cmdInsertDebugMarker(const commands::InsertDebugMarker& cmd);
     void cmdWriteTimestamp(const commands::WriteTimestamp& cmd);
     void cmdExecuteCallback(const commands::ExecuteCallback& cmd);
+    void cmdBuildClusterAccelerationStructure(const commands::BuildClusterAccelerationStructure& cmd);
```

Comment:
Can we move this up to before `cmdConvertCooperativeVectorMatrix` so that all raytracing commands are in the same place?